### DISCUSSION TITLE
Stable build?

### DISCUSSION THREAD

**Discussion by y0himba** [tannerhelland/PhotoDemon#357] [Q&A] [answered]
I'm kind of a Github newb.  I get the notifications for the nightly build, but I also would like to find the stable releases.  Help me gurus!

**Comment by tannerhelland** [comment, marked as answer]
Hi @y0himba !  Apologies if I've misunderstood your question, but PhotoDemon's stable releases are very infrequent (maybe once a year).  The last stable release was version 8.4 from last year, available from photodemon.org or the top of PhotoDemon's GitHub releases page:

https://github.com/tannerhelland/PhotoDemon/releases

If you were asking about GitHub notifications for stable releases, unfortunately, GitHub does not currently provide granular control over release notification.  You can only be notified of "any releases" or no releases (as far as I know).

Looks like some people have petitioned GitHub for more control over this:

https://github.com/isaacs/github/issues/410

PhotoDemon itself provides more control over update notifications, if that helps.  In the `Tools > Options > Updates` menu, you can set notifications to include stable, beta, or developer build updates:

![image](https://user-images.githubusercontent.com/1930029/126673936-efdd52f1-e003-4a7c-91dc-bfe7bdd1317b.png)

Doesn't help with GitHub notifications, I realize, but maybe better than nothing!

**Comment by y0himba** [comment]
You understood me correctly and answered my question.  Thank you!
